joomla/joomla-cms
# Improve installer warning

### Summary of Changes
Give more information when the installation of an extension fails.
The german Joomla forum is full with quetions about the message "Can't find XML setup file" and supporters always write the same answer: Check your tmp folder and permissions.

So the Warning can contain this information. 

### Testing Instructions
Code inspect or 
Rename the tmp folder in one of your applications and see the message when you install an extension before and after the patch.

Native speakers: please help with correct English. 
Of course there can be other reasons for this warning, a more generic wording would be appreciated.
